src/types/index.d.ts

```diff
@@ -567,10 +567,18 @@ export interface ZoneEgressOverview extends MeshEntity {
   zoneEgressInsight: any
 }
 
+export interface DpCert {
+  rotation?: {
+    expiration?: string
+  }
+  requestTimeout?: string
+}
+
 export interface Backend {
   name: string
   type: string
-  conf?: Record<string, string | number>
+  conf?: Conf
```

Comment:
How come we went from `Record<string, string | number>` to `object`?

Reply:
Because that type, it turns out, isn’t correct. The `conf` here is very similar to the conf field we find on, for example, the _rules API. It can contain a bunch of things including nested objects (which the `Record<string, string | number>` type doesn’t allow).

I noticed that while adding the tests: the type checking there told me my input had unexpected properties.

Reply:
Doesn't `object` include `() => {}`? Should this be `Record<string, unknown>`?

Reply:
You’re right. I changed the `Conf` type to be `Record<string, unknown>`.